packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1443,6 +1443,7 @@ export interface CommandProperties {
     };
 }
 
+export type TaskGroupKind = 'build' | 'test' | 'rebuild' | 'clean'
```

Comment:
```suggestion
export type TaskGroupKind = 'build' | 'test' | 'rebuild' | 'clean';
```

Seems like the linter requires a semicolon here.